sunpy/net/dataretriever/tests/test_stereo.py

```diff
@@ -0,0 +1,70 @@
+"""
+This module tests STEREO CLient.
+"""
+#This module was developed with funding provided by
+#the Google Summer of Code 2016.
+import pytest
+
+from sunpy.time.timerange import TimeRange
+from sunpy.net.vso.attrs import Time, Instrument, Source, Detector
+from sunpy.net.dataretriever.client import QueryResponse
+from sunpy.net.dataretriever.downloader_factory import UnifiedResponse
+from sunpy.net import Fido
+from sunpy.net import attrs as a
+import sunpy.net.dataretriever.sources.stereo as stereo
+
+SECCHIClient = stereo.SECCHIClient()
+PClient = stereo.PLASTICClient()
+IClient = stereo.IMPACTClient()
+SWClient = stereo.SWAVESClient()
+
+@pytest.mark.online
+@pytest.mark.parametrize("timerange, source, detector, url_start, url_end",
+                         [(TimeRange('2008/3/2 17:45:00', '2008/3/2 18:00:00'), 'STEREO_A', 'euvi',
+                          'http://stereo-ssc.nascom.nasa.gov/data/beacon/ahead/secchi/img/euvi/20080302/20080302_174530_n7euA.fts',
+                         'http://stereo-ssc.nascom.nasa.gov/data/beacon/ahead/secchi/img/euvi/20080302/20080302_175530_n7euA.fts')])
+def test_get_url_for_timerange(timerange, source, detector, url_start, url_end):
+    urls = SECCHIClient._get_url_for_timerange(timerange, source=source, detector=detector)
+    assert isinstance(urls, list)
+    assert urls[0] == url_start
+    assert urls[-1] == url_end
+
+trange = Time('2008/3/2 17:45:00', '2008/3/2 18:00:00')
```

Comment:
if this `trange` is just for the function below, then put it inside the function.